ZendeskApiRepl/Program.cs

```diff
@@ -0,0 +1,28 @@
+// See https://aka.ms/new-console-template for more information
```

Comment:
I personally feel that the integration tests you have added serve as adequate documentation. Do you still see value in this?

Reply:
yeah good point, thanks! I have removed it and added the version number.